OMGVA PoS/docker/mssql-db/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+services:
+  mssql:
+    build: .
+    container_name: mssql_container
+    ports:
+      - "1433:1433"
+    environment:
+      - ACCEPT_EULA=Y
+      - MSSQL_SA_PASSWORD=omgva_POS_999
```

Comment:
You put the password into github

Reply:
Since this is a learning project and it's a password for local env I think it's not a problem and makes setup easier. Wanted to make a simpler, more 'default' password, however mssql required a better one

However, good catch !